pkg/volume/csi/csi_mounter.go

```diff
@@ -108,49 +110,93 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
-	csiSource, err := getCSISourceFromSpec(c.spec)
-	if err != nil {
-		klog.Error(log("mounter.SetupAt failed to get CSI persistent source: %v", err))
-		return err
-	}
-
 	csi := c.csiClient
 	ctx, cancel := context.WithTimeout(context.Background(), csiTimeout)
 	defer cancel()
 
-	// Check for STAGE_UNSTAGE_VOLUME set and populate deviceMountPath if so
-	deviceMountPath := ""
-	stageUnstageSet, err := csi.NodeSupportsStageUnstage(ctx)
+	volSrc, pvSrc, err := getSourceFromSpec(c.spec)
 	if err != nil {
-		klog.Error(log("mounter.SetUpAt failed to check for STAGE_UNSTAGE_VOLUME capabilty: %v", err))
+		klog.Error(log("mounter.SetupAt failed to get CSI persistent source: %v", err))
```

Comment:
I prefer not to log errors that are also going to be returned. It causes redundant logs to be emitted.

Reply:
I missed that one. Will fix next.